docs/products/flink/getting-started.rst

```diff
@@ -1,12 +1,74 @@
 Getting started
 ===============
 
-Aiven for Flink is available in the `Aiven console <https://console.aiven.io>`_.
+The first steps in using Aiven for Apache Flink are to create a service and a Flink job. You can do this in the Aiven web console or with the Aiven CLI.
 
-Choose the Apache Flink version, your cloud provider and location to deploy to, then choose which plan to use.
+In addition to the Flink service, you also need a service that you can use as a data source. This article outlines an example of using an Apache Kafka service as the data source.
 
-Finally, give the service a name and then select "Create Service", and your new Flink service will start building. While it does that, you can already visit the service overview page to see the details of the service.
 
-.. image:: /images/products/flink/console-overview.png
+Create an Apache Flink service in the Aiven web console
+-------------------------------------------------------
 
 
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
```

Comment:
I believe we can skip this description, there is no interaction needed from the user point of view and the new page is pretty self-explanatory

Reply:
True, that's now removed ;)